### ISSUE TITLE
Missmatch issue

### ISSUE THREAD

**Issue by srx3001** [griffithlab/pVACtools#150]
Hello, I am working with VCF data called with GATK using hg19.
Now, when using VEP, I use the Grch37 reference (number 174), as they should be identical minus the MT genes.

I run VEP with the following commands:

vep --input_file SMT_processed.vcf.gz --format vcf --output_file SM.vcf --vcf --symbol --terms SO --plugin Downstream --plugin Wildtype --cache 174 --merged --offline

I get the output without errors.
Then when I run PVAC with:

pvacseq run SM.vcf SM HLA-G*01:09,HLA-E*01:01,H2-IAb NetMHC PickPocket NNalign SMTOOLS/ -e 9,10 --net-chop-method cterm --netmhc-stab --top-score-metric=lowest -d full --keep-tmp-files

I get a few of these warnings/errors:

Wildtype sequence length is shorter than desired peptide sequence length at position (chr3, 154801977, 154801978). Using wildtype sequence length (15) instead.

and these:

frameshift_variant transcript does not contain a DownstreamProtein sequence. Skipping.
chr3 130733054 TTTTG T NM_014065.3

In the end it breaks with the following error:

ERROR: There was a mismatch between the actual wildtype amino acid sequence (QQQQQQQQQQHSNQTSNWSP) and the expected amino acid sequence (QQQQQQQQQQQQQQQQQQQQ). Did you use the same reference build version for VEP that you used for creating the VCF?
{'reference': 'TTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTG', 'gene_expression': 'NA', 'chromosome_name': 'chr4', 'trna_vaf': 'NA', 'tdna_vaf': 'NA', 'index': '25533.MAML3.ENST00000398940.inframe_del.29-37QQQQQQQQQQQQQQQQQQQQ/QQQQQQQQQQQQQQQQ', 'protein_position': '29-37', 'tdna_depth': 'NA', 'normal_vaf': 'NA', 'transcript_expression': 'NA', 'ensembl_gene_id': 'ENSG00000196782', 'gene_name': 'MAML3', 'variant_type': 'inframe_del', 'variant': 'TTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTGCTG', 'trna_depth': 'NA', 'wildtype_amino_acid_sequence': 'MSPAEQLKQMAAQQQQRAKLMQQKQQQQQQQQQQQQQQHSNQTSNWSPLGPPSSPYGAAFTAEKPNSPMMYPQAFNNQNPIVPPMANNLQKTTMNNYLPQNHMNMINQQPNNLGTNSLNKQHNILTYGNTKPLTHFNADLSQRMTPPVANPNKNPLMPYIQQQQQQQQQQQQQQQQQQPPPPQLQAPRAHLSEDQKRLLLMKQKGVMNQPMAYAALPSHGQVSVKVTHSSSSSCGAFDCRCVMFYKSTVVGSNCLQKTSLHPQVVARFGVFFFHFS', 'normal_depth': 'NA', 'protein_length_change': '', 'transcript_name': 'ENST00000398940', 'amino_acid_change': 'QQQQQQQQQQQQQQQQQQQQ/QQQQQQQQQQQQQQQQ', 'downstream_amino_acid_sequence': '', 'fusion_amino_acid_sequence': '', 'stop': '140811122', 'start': '140811063'}

Is it due to the miss-match between hg19 and my grch37 cache?
Anything else I could use with VEP to alleviate this?

**Comment by susannasiebert** [comment]
Which version of the Wildtype plugins are you using? The newest one can be found here https://github.com/griffithlab/pVACtools/blob/master/tools/pvacseq/VEP_plugins/Wildtype.pm.

**Comment by srx3001** [comment]
I already had the newest one.
I found out that some errors disappear if I use --fasta and supply the hg19.
Still strange they do not match... Ill post more if I hit one of these errors.

**Comment by susannasiebert** [comment]
Ok, in that case I would suggest to go to the Grch37 that is used by VEP and compare the reference at that position to the reference in your hg19 build used for alignment/variant calling. This error would indicate that they are not identical at that position.

**Comment by susannasiebert** [comment]
Closing this issue, since it's not a bug with pVACseq.

**Comment by 41526** [comment]
I have encountered a similar problem.

**Comment by 41526** [comment]
VEP code

`./variant_effect_predictor.pl --input_file /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/WESSRA/WESFastq/_AfterFASTQC_/_AfterAlign_/BAMFile/VEPtest/SMC_OS_42_SomaticTwiceFiltered.vcf.gz --output_file /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/WESSRA/WESFastq/_AfterFASTQC_/_AfterAlign_/BAMFile/VEPtest/VEPoutput/SMC_OS_42VEPed.vcf --format vcf --vcf --symbol --terms SO --tsl --hgvs --fasta /IJob/J33/HLA_example_temp/temp/Sequence/reference.hg19.fa --offline --dir_cache /pub5/xiaoyun/BioSoftware/Conda/envs/VEP/share/variant-effect-predictor-87-1/cache/V87  --plugin Downstream --plugin Wildtype`

pVACseq code

`pvacseq run -e 8,9,10,11 --iedb-install-directory /pub6/temp/pb/Software/IEDB --normal-sample-name SRR5357710 --pass-only /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/WESSRA/WESFastq/_AfterFASTQC_/_AfterAlign_/BAMFile/inputpreprocess/addexpression/SMC_OS_42_AddExp.vcf SRR5357744 HLA-A*02:01,HLA-B*35:01,DRB1*11:01 NNalign NetMHC NetMHCIIpan NetMHCcons NetMHCpan PickPocket SMM SMMPMBEC SMMalign /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/WESSRA/WESFastq/_AfterFASTQC_/_AfterAlign_/BAMFile/pvacseq_output`

Code for obtaining gene expression data from RNAseq data

`RSEM
Make index
rsem-prepare-reference -gtf rsem-prepare-reference -gtf /IJob/J33/HLA_example_temp/temp/Sequence/gencode.v19.annotation.gtf --star /IJob/J33/HLA_example_temp/temp/Sequence/reference.hg19.fa /IJob/J33/HLA_example_temp/temp/Sequence/RSEMindex
Count Expression
rsem-calculate-expression --paired-end -p 6 --star --append-names --output-genome-bam /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/BRCA/3.trimGalore_fastqdata/trimGalore_result/4/trimGalore_result/SRR5357781_1_val_1.fq /IJob/J33/Data/SRP102119_BC_12P22M34NSamples_WES/BRCA/3.trimGalore_fastqdata/trimGalore_result/4/trimGalore_result/SRR5357781_2_val_2.fq /IJob/J33/HLA_example_temp/temp/Sequence/RSEMindex /pub6/temp/pb/SRP102119_BC_12P22M34NSamples_RNAseqhg19STAR/expressionfile/SRR5357781
`

I am about to collapse.

By the way, I can't use the latest version of pvactools, even if I install TensorFlow with conda. The anomaly is that no matter what command you execute with pvacseq, the terminal returns nothing. However, when `pvactools -v` is executed, the version information of pvactools can be displayed very well.

**Comment by susannasiebert** [comment]
Which reference fasta did you use when calling variants? Is the same that you used for running VEP?

Your VEP cache might also be incompatible with your reference.

Can you please post the exact error you get while running pVACseq?

Regarding using the latest version of pVACtools: There is a bug in the 1.3.2 release that results in error messages getting suppressed. I just made a new version that should solve this problem. Please upgrade to version 1.3.3 (pip install pvactools --upgrade). You should then see what exactly the error is. Most likely it is still related to your installation of Tensorflow. Please post you're error and I will be happy to help you debug the problem.

**Comment by 41526** [comment]
The reference sequence file used to get the VCF is the same file as the reference sequence file used to run VEP, hg19.

My VEP cache file is homo_sapiens_vep_87_GRCh37.tar.gz.

I now suspect that the small_exac_common_3.hg19.vcf.gz file I used to filter VCF files with GATK was obtained by liftover conversion from small_exac_common_3.hg38.vcf.gz, so I caused a mismatch error. I am trying to use unfiltered The VCF calculates neoantigen, hoping there will be no mismatch errors.

**Comment by 41526** [comment]
I upgraded pvactools to 1.3.3 and executed the `pip uninstall tensorflow` & `conda install tensorflow==1.8.0` terminal showing All requested packages already installed.

But when I run pvacseq run -h terminal it shows ImportError: No module named 'tensorflow'.

The complete error message is as follows:

`Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/pvacseq", line 7, in <module>
    from tools.pvacseq.main import main
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/tools/pvacseq/__init__.py", line 16, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/tools/pvacseq/run.py", line 4, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/lib/__init__.py", line 26, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/lib/binding_filter.py", line 6, in <module>
    from lib.allele_specific_binding_filter import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/lib/allele_specific_binding_filter.py", line 3, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/lib/prediction_class.py", line 13, in <module>
    raise err
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/lib/prediction_class.py", line 10, in <module>
    from mhcnuggets.src.predict import predict
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/mhcnuggets/src/predict.py", line 11, in <module>
    from mhcnuggets.src.models import get_predictions, mhcnuggets_lstm
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/mhcnuggets/src/models.py", line 9, in <module>
    import keras
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/keras/__init__.py", line 3, in <module>
    from . import utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/keras/utils/__init__.py", line 6, in <module>
    from . import conv_utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/keras/utils/conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/keras/backend/__init__.py", line 89, in <module>
    from .tensorflow_backend import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
ImportError: No module named 'tensorflow'`

I know the error message indicates that there is no tensorflow module, but I did install this module, so I was confused.

**Comment by 41526** [comment]
I changed a germline mutation data to filter vcf, the obtained vcf and then VEP annotation, then add gene expression data with vcf-expression-annotator, and finally use pvacseq to get neoantigen, still showing mismatch error message. Now I don't know what to do :-( .

The command parameters are the same as above, only the input files are different.

**Comment by susannasiebert** [comment]
My hunch is that there are slight differences between your hg19 reference and the build 37 VEP cache, which might be causing these errors. Without knowing the exact locations that are causing these problems for you, I won't be able to say for sure. You could remove the variants that are throwing errors in order to at least get a successful pVACseq run. You could also try updating VEP to a new version and re-annotating your VCF to see if a newer VEP version fixes this issue.

**Comment by susannasiebert** [comment]
Regarding tensorflow, what does `pip show tensorflow` return?

**Comment by 41526** [comment]
I executed pip show tensorflow and nothing return.

I am trying to reinstall VEP via git clone, hopefully it will help me.

**Comment by susannasiebert** [comment]
If pip show returns nothing then tensorflow is not installed correctly. Try `pip install tensorflow==1.8.0 --upgrade --force-reinstall`.

**Comment by 41526** [comment]
`(pVACTools) [wanly2018@admin ~]$ pip show tensorflow
Name: tensorflow
Version: 1.8.0
Summary: TensorFlow helps the tensors flow
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: opensource@google.com
License: Apache 2.0
Location: /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages
Requires: wheel, tensorboard, gast, six, protobuf, termcolor, grpcio, astor, absl-py, numpy
Required-by: pvactools, mhcnuggets, mhcflurry`

`(pVACTools) [wanly2018@admin ~]$ pvacseq run -h
Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/pvacseq", line 6, in <module>
    from tools.pvacseq.main import main
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/__init__.py", line 16, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/run.py", line 4, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/__init__.py", line 26, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/binding_filter.py", line 6, in <module>
    from lib.allele_specific_binding_filter import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/allele_specific_binding_filter.py", line 3, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 13, in <module>
    raise err
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 10, in <module>
    from mhcnuggets.src.predict import predict
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/predict.py", line 11, in <module>
    from mhcnuggets.src.models import get_predictions, mhcnuggets_lstm
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/models.py", line 9, in <module>
    import keras
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/__init__.py", line 3, in <module>
    from . import utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/__init__.py", line 6, in <module>
    from . import conv_utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/__init__.py", line 89, in <module>
    from .tensorflow_backend import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.`

I am desperate

**Comment by susannasiebert** [comment]
Ok, so that looks like the same error you had previously. What happens when you run `pip uninstall tensorflow` and `conda install tensorflow==1.8.0 --force`?

**Comment by 41526** [comment]
What you asked me to try was CONDA install tensorflow == 1.8.0 -- force. Did you write the command incorrectly?

**Comment by 41526** [comment]
First, I executed the `pip uninstall tensorflow` command, and then `conda install tensorflow == 1.8.0 -- force`.
`pip show tensorflow`
`
Name: tensorflow
Version: 1.8.0
Summary: TensorFlow helps the tensors flow
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: opensource@google.com
License: Apache 2.0
Location: /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages
Requires: gast, numpy, grpcio, absl-py, tensorboard, wheel, six, termcolor, astor, protobuf

Required-by: pvactools, mhcnuggets, mhcflurry
`
Eventually I executed the `pvacseq run-h` command
And the error message is the same.

`Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/pvacseq", line 6, in <module>
    from tools.pvacseq.main import main
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/__init__.py", line 16, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/run.py", line 4, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/__init__.py", line 26, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/binding_filter.py", line 6, in <module>
    from lib.allele_specific_binding_filter import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/allele_specific_binding_filter.py", line 3, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 13, in <module>
    raise err
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 10, in <module>
    from mhcnuggets.src.predict import predict
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/predict.py", line 11, in <module>
    from mhcnuggets.src.models import get_predictions, mhcnuggets_lstm
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/models.py", line 9, in <module>
    import keras
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/__init__.py", line 3, in <module>
    from . import utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/__init__.py", line 6, in <module>
    from . import conv_utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/__init__.py", line 89, in <module>
    from .tensorflow_backend import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.`

**Comment by susannasiebert** [comment]
Hm, that is strange. If anything, expected to see the same `ImportError: No module named 'tensorflow'` you got before, not the `GLIBC_2.16` error. Are you sure `pip uninstall tensorflow` and `conda install tensorflow == 1.8.0 --force` executed correctly? Can you please post the output from when you ran these two steps?

**Comment by 41526** [comment]
`pip uninstall tensorflow`

(pVACTools) [wanly2018@admin ~]$ pip uninstall tensorflow
Uninstalling tensorflow-1.8.0:
  Would remove:
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/freeze_graph
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/saved_model_cli
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/tensorboard
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/toco
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/toco_from_protos
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/external/*
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow-1.8.0.dist-info/*
    /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/*
Proceed (y/n)? y
  Successfully uninstalled tensorflow-1.8.0

`conda install tensorflow == 1.8.0 --force`

(pVACTools) [wanly2018@admin ~]$ conda install tensorflow==1.8.0 --force


WARNING: The --force flag will be removed in a future conda release.
         See 'conda install --help' for details about the --force-reinstall
         and --clobber flags.


Collecting package metadata: done
Solving environment: | 
Warning: 4 possible package resolutions (only showing differing packages):
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py36_0
  - https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py36_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py36_0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py36_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2ldone


==> WARNING: A newer version of conda exists. <==
  current version: 4.6.7
  latest version: 4.6.8

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools

  added / updated specs:
    - tensorflow==1.8.0



Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done

`pip show tensorflow`

(pVACTools) [wanly2018@admin ~]$ pip show tensorflow
Name: tensorflow
Version: 1.8.0
Summary: TensorFlow helps the tensors flow
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: opensource@google.com
License: Apache 2.0
Location: /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages
Requires: six, astor, termcolor, absl-py, numpy, grpcio, tensorboard, wheel, protobuf, gast
Required-by: pvactools, mhcnuggets, mhcflurry

`pvacseq run -h`

(pVACTools) [wanly2018@admin ~]$ pvacseq run -h
Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/bin/pvacseq", line 6, in <module>
    from tools.pvacseq.main import main
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/__init__.py", line 16, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tools/pvacseq/run.py", line 4, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/__init__.py", line 26, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/binding_filter.py", line 6, in <module>
    from lib.allele_specific_binding_filter import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/allele_specific_binding_filter.py", line 3, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 13, in <module>
    raise err
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/lib/prediction_class.py", line 10, in <module>
    from mhcnuggets.src.predict import predict
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/predict.py", line 11, in <module>
    from mhcnuggets.src.models import get_predictions, mhcnuggets_lstm
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/mhcnuggets/src/models.py", line 9, in <module>
    import keras
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/__init__.py", line 3, in <module>
    from . import utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/__init__.py", line 6, in <module>
    from . import conv_utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/utils/conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/__init__.py", line 89, in <module>
    from .tensorflow_backend import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools/lib/python3.6/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.

**Comment by susannasiebert** [comment]
At this point I would try and start with a fresh conda environment:
```
conda create -n pVACTools_new python=3.5 tensorflow=1.8.0
conda activate pVACTools_new
pip install pvactools==1.3.3
pvacseq -h
```

**Comment by 41526** [comment]
`conda create -n pVACTools_new python=3.5 tensorflow=1.8.0`

[wanly2018@admin ~]$ conda create -n pVACTools_new python=3.5 tensorflow=1.8.0
Collecting package metadata: done
Solving environment: \ 
Warning: >10 possible package resolutions (only showing differing packages):
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0
  - https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::readline-6.2-2
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0, https://mirrors.ustc.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::certifi-2016.2.28-py35_0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::openssl-1.0.2l-0, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::readline-6.2-2, https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free::zlib-1.2.11-0
  ... and othedone


==> WARNING: A newer version of conda exists. <==
  current version: 4.6.7
  latest version: 4.6.8

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new

  added / updated specs:
    - python=3.5
    - tensorflow=1.8.0


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    openssl-1.0.2l             |                0         3.2 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
    pip-9.0.1                  |           py35_1         1.7 MB  https://mirrors.ustc.edu.cn/anaconda/pkgs/free
    readline-6.2               |                2         606 KB  https://mirrors.ustc.edu.cn/anaconda/pkgs/free
    setuptools-36.4.0          |           py35_1         567 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free
    wheel-0.29.0               |           py35_0          82 KB  https://mirrors.ustc.edu.cn/anaconda/pkgs/free
    ------------------------------------------------------------
                                           Total:         6.1 MB

The following NEW packages will be INSTALLED:

  absl-py            anaconda/cloud/conda-forge/noarch::absl-py-0.5.0-py_0
  astor              anaconda/cloud/conda-forge/noarch::astor-0.7.1-py_0
  blas               anaconda/pkgs/free/linux-64::blas-1.0-mkl
  bleach             anaconda/pkgs/free/linux-64::bleach-1.5.0-py35_0
  certifi            anaconda/pkgs/free/linux-64::certifi-2016.2.28-py35_0
  gast               anaconda/cloud/conda-forge/noarch::gast-0.2.2-py_0
  grpcio             pkgs/main/linux-64::grpcio-1.12.1-py35hdbcaa40_0
  html5lib           anaconda/pkgs/free/linux-64::html5lib-0.9999999-py35_0
  libgcc-ng          anaconda/cloud/conda-forge/linux-64::libgcc-ng-7.3.0-hdf63c60_0
  libgfortran-ng     anaconda/cloud/conda-forge/linux-64::libgfortran-ng-7.2.0-hdf63c60_3
  libprotobuf        anaconda/pkgs/free/linux-64::libprotobuf-3.4.0-0
  libstdcxx-ng       anaconda/cloud/conda-forge/linux-64::libstdcxx-ng-7.3.0-hdf63c60_0
  markdown           anaconda/pkgs/free/linux-64::markdown-2.6.9-py35_0
  mkl                anaconda/pkgs/free/linux-64::mkl-2017.0.3-0
  numpy              pkgs/main/linux-64::numpy-1.14.2-py35hdbf6ddf_0
  openssl            anaconda/pkgs/free/linux-64::openssl-1.0.2l-0
  pip                anaconda/pkgs/free/linux-64::pip-9.0.1-py35_1
  protobuf           anaconda/pkgs/free/linux-64::protobuf-3.4.0-py35_0
  python             anaconda/pkgs/free/linux-64::python-3.5.4-0
  readline           anaconda/pkgs/free/linux-64::readline-6.2-2
  setuptools         anaconda/pkgs/free/linux-64::setuptools-36.4.0-py35_1
  six                anaconda/pkgs/free/linux-64::six-1.10.0-py35_0
  sqlite             anaconda/pkgs/free/linux-64::sqlite-3.13.0-0
  tensorboard        anaconda/cloud/conda-forge/linux-64::tensorboard-1.8.0-py35_1
  tensorflow         anaconda/cloud/conda-forge/linux-64::tensorflow-1.8.0-py35_1
  termcolor          anaconda/pkgs/free/linux-64::termcolor-1.1.0-py35_0
  tk                 anaconda/pkgs/free/linux-64::tk-8.5.18-0
  werkzeug           anaconda/pkgs/free/linux-64::werkzeug-0.12.2-py35_0
  wheel              anaconda/pkgs/free/linux-64::wheel-0.29.0-py35_0
  xz                 anaconda/pkgs/free/linux-64::xz-5.2.3-0
  zlib               anaconda/pkgs/free/linux-64::zlib-1.2.11-0


Proceed ([y]/n)? y


Downloading and Extracting Packages
openssl-1.0.2l       | 3.2 MB    | ######################################################################################## | 100% 
setuptools-36.4.0    | 567 KB    | ######################################################################################## | 100% 
readline-6.2         | 606 KB    | ######################################################################################## | 100% 
pip-9.0.1            | 1.7 MB    | ######################################################################################## | 100% 
wheel-0.29.0         | 82 KB     | ######################################################################################## | 100% 
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use:
# > source activate pVACTools_new
#
# To deactivate an active environment, use:
# > source deactivate
#

`conda activate pVACTools_new`

[wanly2018@admin ~]$ source activate pVACTools_new
(pVACTools_new) [wanly2018@admin ~]$ pip install pvactools==1.3.3

`pip install pvactools==1.3.3`

(pVACTools_new) [wanly2018@admin ~]$ pip install pvactools==1.3.3
Collecting pvactools==1.3.3
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/2b/e4/284565ea9caa6da31a4ad6e343be3960549d45e5abe9156c4a3089a917b8/pvactools-1.3.3-py3-none-any.whl
Collecting biopython (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/3c/5a/05c29e3a75269e00e50779b28b60041d9783ee79054ce53a1d193e05deab/biopython-1.73-cp35-cp35m-manylinux1_x86_64.whl
Collecting requests (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/7d/e3/20f3d364d6c8e5d2353c72a67778eb189176f08e873c9900e10c0287b84b/requests-2.21.0-py2.py3-none-any.whl
Requirement already satisfied: tensorflow==1.8.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from pvactools==1.3.3)
Collecting Pillow (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/8b/e9/5c47710fe383f0582da668302a80a6355fe15c2ce2dde89b50fe34acefa6/Pillow-5.4.1-cp35-cp35m-manylinux1_x86_64.whl
Collecting py-postgresql (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting jsonschema==2.6.0 (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/77/de/47e35a97b2b05c2fadbec67d44cfcdcd09b8086951b331d82de90d2912da/jsonschema-2.6.0-py2.py3-none-any.whl
Collecting watchdog (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting simanneal (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/b4/63/0cfaa737d94a273295b93d1b80498d2db7ce9004e2e53312b10149cd6006/simanneal-0.4.2-py2.py3-none-any.whl
Collecting networkx (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting mhcnuggets (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting pymp-pypi (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting bokeh==0.13.0 (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting connexion==1.4.2 (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/9f/1b/5e509fd2921827aaa7a5f39530593b95667c9c8da8289628d42efaf7432c/connexion-1.4.2-py2.py3-none-any.whl
Collecting mhcflurry (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting PyYAML (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting PyVCF (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting pandas (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/74/24/0cdbf8907e1e3bc5a8da03345c23cbed7044330bb8f73bb12e711a640a00/pandas-0.24.2-cp35-cp35m-manylinux1_x86_64.whl (10.0MB)
    100% |████████████████████████████████| 10.0MB 37kB/s 
Collecting wget (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting swagger-spec-validator==2.1.0 (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/44/02/bcc0122d561d9727b8ca476058f2c57a37a1c86d0f7c9aec5543f3219cd0/swagger_spec_validator-2.1.0-py2.py3-none-any.whl
Collecting pysam (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/aa/e6/0b18d30c30e3690c5774446c1d7246c5e37c4710e38d93a20bb56cad4bc0/pysam-0.15.2-cp35-cp35m-manylinux1_x86_64.whl
Collecting flask-cors (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/65/cb/683f71ff8daa3aea0a5cbb276074de39f9ab66d3fbb8ad5efb5bb83e90d2/Flask_Cors-3.0.7-py2.py3-none-any.whl
Collecting tornado==5.0.2 (from pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Requirement already satisfied: numpy in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from biopython->pvactools==1.3.3)
Collecting certifi>=2017.4.17 (from requests->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/60/75/f692a584e85b7eaba0e03827b3d51f45f571c2e793dd731e598828d380aa/certifi-2019.3.9-py2.py3-none-any.whl
Collecting urllib3<1.25,>=1.21.1 (from requests->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/62/00/ee1d7de624db8ba7090d1226aebefab96a2c71cd5cfa7629d6ad3f61b79e/urllib3-1.24.1-py2.py3-none-any.whl
Collecting chardet<3.1.0,>=3.0.2 (from requests->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl
Collecting idna<2.9,>=2.5 (from requests->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/14/2c/cd551d81dbe15200be1cf41cd03869a46fe7226e7450af7a6545bfc474c9/idna-2.8-py2.py3-none-any.whl
Requirement already satisfied: astor>=0.6.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: six>=1.10.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: termcolor>=1.1.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: absl-py>=0.1.6 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: wheel>=0.26 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: grpcio>=1.8.6 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: gast>=0.2.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: tensorboard<1.9.0,>=1.8.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: protobuf>=3.4.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorflow==1.8.0->pvactools==1.3.3)
Collecting pathtools>=0.1.1 (from watchdog->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting argh>=0.24.1 (from watchdog->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/06/1c/e667a7126f0b84aaa1c56844337bf0ac12445d1beb9c8a6199a7314944bf/argh-0.26.2-py2.py3-none-any.whl
Collecting decorator>=4.3.0 (from networkx->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Downloading https://files.pythonhosted.org/packages/5f/88/0075e461560a1e750a0dcbf77f1d9de775028c37a19a346a6c565a257399/decorator-4.4.0-py2.py3-none-any.whl
Collecting keras (from mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/5e/10/aa32dad071ce52b5502266b5c659451cfd6ffcbf14e6c8c4f16c0ff5aaab/Keras-2.2.4-py2.py3-none-any.whl
Collecting scipy (from mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/f0/30/526bee2ce18c066f9ff13ba89603f6c2b96c9fd406b57a21a7ba14bf5679/scipy-1.2.1-cp35-cp35m-manylinux1_x86_64.whl
Collecting scikit-learn (from mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/38/7f/7d21bd89a97603a14d3770ca912d4402c394a8e50642f9b8b5cc6867a61a/scikit_learn-0.20.3-cp35-cp35m-manylinux1_x86_64.whl
Collecting packaging>=16.8 (from bokeh==0.13.0->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/91/32/58bc30e646e55eab8b21abf89e353f59c0cc02c417e42929f4a9546e1b1d/packaging-19.0-py2.py3-none-any.whl
Collecting python-dateutil>=2.1 (from bokeh==0.13.0->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/41/17/c62faccbfbd163c7f57f3844689e3a78bae1f403648a6afb1d0866d87fbb/python_dateutil-2.8.0-py2.py3-none-any.whl
Collecting Jinja2>=2.7 (from bokeh==0.13.0->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/7f/ff/ae64bacdfc95f27a016a7bed8e8686763ba4d277a78ca76f32659220a731/Jinja2-2.10-py2.py3-none-any.whl
Collecting inflection>=0.3.1 (from connexion==1.4.2->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting clickclick>=1.2 (from connexion==1.4.2->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/b6/51/2b04f7a56dcbacc0e3a7cf726e1d88d28866bf488a7a0668582306e1e643/clickclick-1.2.2-py2.py3-none-any.whl
Collecting typing>=3.6.1; python_version < "3.6" (from connexion==1.4.2->pvactools==1.3.3)
  Using cached https://files.pythonhosted.org/packages/4a/bd/eee1157fc2d8514970b345d69cb9975dcd1e42cd7e61146ed841f6e68309/typing-3.6.6-py3-none-any.whl
Collecting flask>=0.10.1 (from connexion==1.4.2->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/7f/e7/08578774ed4536d3242b14dacb4696386634607af824ea997202cd0edb4b/Flask-1.0.2-py2.py3-none-any.whl
Collecting mhcnames (from mhcflurry->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
Collecting appdirs (from mhcflurry->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Collecting tqdm (from mhcflurry->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/6c/4b/c38b5144cf167c4f52288517436ccafefe9dc01b8d1c190e18a6b154cd4a/tqdm-4.31.1-py2.py3-none-any.whl
Requirement already satisfied: setuptools in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from PyVCF->pvactools==1.3.3)
Collecting pytz>=2011k (from pandas->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/61/28/1d3920e4d1d50b19bc5d24398a7cd85cc7b9a75a490570d5a30c57622d34/pytz-2018.9-py2.py3-none-any.whl
Requirement already satisfied: bleach==1.5.0 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorboard<1.9.0,>=1.8.0->tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: html5lib==0.9999999 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorboard<1.9.0,>=1.8.0->tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: werkzeug>=0.11.10 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorboard<1.9.0,>=1.8.0->tensorflow==1.8.0->pvactools==1.3.3)
Requirement already satisfied: markdown>=2.6.8 in /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages (from tensorboard<1.9.0,>=1.8.0->tensorflow==1.8.0->pvactools==1.3.3)
Collecting keras-preprocessing>=1.0.5 (from keras->mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/c0/bf/0315ef6a9fd3fc2346e85b0ff1f5f83ca17073f2c31ac719ab2e4da0d4a3/Keras_Preprocessing-1.0.9-py2.py3-none-any.whl
Collecting h5py (from keras->mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/4c/77/c4933e12dca0f61bcdafc207c7532e1250b8d12719459fd85132f3daa9fd/h5py-2.9.0-cp35-cp35m-manylinux1_x86_64.whl
Collecting keras-applications>=1.0.6 (from keras->mhcnuggets->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/90/85/64c82949765cfb246bbdaf5aca2d55f400f792655927a017710a78445def/Keras_Applications-1.0.7-py2.py3-none-any.whl
Collecting pyparsing>=2.0.2 (from packaging>=16.8->bokeh==0.13.0->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/de/0a/001be530836743d8be6c2d85069f46fecf84ac6c18c7f5fb8125ee11d854/pyparsing-2.3.1-py2.py3-none-any.whl
Collecting MarkupSafe>=0.23 (from Jinja2>=2.7->bokeh==0.13.0->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/6e/57/d40124076756c19ff2269678de7ae25a14ebbb3f6314eb5ce9477f191350/MarkupSafe-1.1.1-cp35-cp35m-manylinux1_x86_64.whl
Collecting click>=4.0 (from clickclick>=1.2->connexion==1.4.2->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl
Collecting itsdangerous>=0.24 (from flask>=0.10.1->connexion==1.4.2->pvactools==1.3.3)
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/76/ae/44b03b253d6fade317f32c24d100b3b35c2239807046a4c953c7b89fa49e/itsdangerous-1.1.0-py2.py3-none-any.whl
Installing collected packages: biopython, certifi, urllib3, chardet, idna, requests, Pillow, py-postgresql, jsonschema, pathtools, argh, PyYAML, watchdog, simanneal, decorator, networkx, keras-preprocessing, scipy, h5py, keras-applications, keras, scikit-learn, mhcnuggets, pymp-pypi, pyparsing, packaging, python-dateutil, MarkupSafe, Jinja2, tornado, bokeh, swagger-spec-validator, inflection, click, clickclick, typing, itsdangerous, flask, connexion, mhcnames, pytz, pandas, appdirs, tqdm, mhcflurry, PyVCF, wget, pysam, flask-cors, pvactools
  Found existing installation: certifi 2016.2.28
    Uninstalling certifi-2016.2.28:
      Successfully uninstalled certifi-2016.2.28
Successfully installed Jinja2-2.10 MarkupSafe-1.1.1 Pillow-5.4.1 PyVCF-0.6.8 PyYAML-5.1 appdirs-1.4.3 argh-0.26.2 biopython-1.73 bokeh-0.13.0 certifi-2019.3.9 chardet-3.0.4 click-7.0 clickclick-1.2.2 connexion-1.4.2 decorator-4.4.0 flask-1.0.2 flask-cors-3.0.7 h5py-2.9.0 idna-2.8 inflection-0.3.1 itsdangerous-1.1.0 jsonschema-2.6.0 keras-2.2.4 keras-applications-1.0.7 keras-preprocessing-1.0.9 mhcflurry-1.2.3 mhcnames-0.4.8 mhcnuggets-2.2 networkx-2.2 packaging-19.0 pandas-0.24.2 pathtools-0.1.2 pvactools-1.3.3 py-postgresql-1.2.1 pymp-pypi-0.4.2 pyparsing-2.3.1 pysam-0.15.2 python-dateutil-2.8.0 pytz-2018.9 requests-2.21.0 scikit-learn-0.20.3 scipy-1.2.1 simanneal-0.4.2 swagger-spec-validator-2.1.0 tornado-5.0.2 tqdm-4.31.1 typing-3.6.6 urllib3-1.24.1 watchdog-0.9.0 wget-3.2
You are using pip version 9.0.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.

`pvacseq -h`

(pVACTools_new) [wanly2018@admin ~]$ pvacseq -h
Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/bin/pvacseq", line 7, in <module>
    from tools.pvacseq.main import main
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tools/pvacseq/__init__.py", line 16, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tools/pvacseq/run.py", line 4, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/lib/__init__.py", line 26, in <module>
    from . import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/lib/binding_filter.py", line 6, in <module>
    from lib.allele_specific_binding_filter import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/lib/allele_specific_binding_filter.py", line 3, in <module>
    from lib.prediction_class import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/lib/prediction_class.py", line 13, in <module>
    raise err
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/lib/prediction_class.py", line 10, in <module>
    from mhcnuggets.src.predict import predict
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/mhcnuggets/src/predict.py", line 11, in <module>
    from mhcnuggets.src.models import get_predictions, mhcnuggets_lstm
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/mhcnuggets/src/models.py", line 9, in <module>
    import keras
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/keras/__init__.py", line 3, in <module>
    from . import utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/keras/utils/__init__.py", line 6, in <module>
    from . import conv_utils
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/keras/utils/conv_utils.py", line 9, in <module>
    from .. import backend as K
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/keras/backend/__init__.py", line 89, in <module>
    from .tensorflow_backend import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/keras/backend/tensorflow_backend.py", line 5, in <module>
    import tensorflow as tf
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/__init__.py", line 24, in <module>
    from tensorflow.python import pywrap_tensorflow  # pylint: disable=unused-import
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/__init__.py", line 49, in <module>
    from tensorflow.python import pywrap_tensorflow
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow.py", line 74, in <module>
    raise ImportError(msg)
ImportError: Traceback (most recent call last):
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow.py", line 58, in <module>
    from tensorflow.python.pywrap_tensorflow_internal import *
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 28, in <module>
    _pywrap_tensorflow_internal = swig_import_helper()
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/pywrap_tensorflow_internal.py", line 24, in swig_import_helper
    _mod = imp.load_module('_pywrap_tensorflow_internal', fp, pathname, description)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/imp.py", line 343, in load_dynamic
    return _load(spec)
ImportError: /usr/lib64/libc.so.6: version `GLIBC_2.16' not found (required by /pub5/xiaoyun/BioSoftware/Conda/envs/pVACTools_new/lib/python3.5/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so)


Failed to load the native TensorFlow runtime.

See https://www.tensorflow.org/install/install_sources#common_installation_problems

for some common reasons and solutions.  Include the entire stack trace
above this error message when asking for help.

: -(

**Comment by susannasiebert** [comment]
I'm confused. https://github.com/griffithlab/pVACtools/issues/314#issuecomment-470754721 indicates that this worked for you in the past. Did something change on your end since then?

Either way, unfortunately, I won't be able to provide additional debugging help since this is a problem with tensorflow, not with pVACtools itself. I suggest commenting on https://github.com/tensorflow/tensorflow/issues/53 or making a new issue in the tensorflow GitHub repo to get help from the tensorflow team. We strongly recommend switching to our Docker image (`griffithlab/pvactools`) to get around system-specific installation issues.

**Comment by 41526** [comment]
Okey, I'll try Docker.

**Comment by 41526** [comment]
Yes, this problem can be solved with this method in pvactools V1.2.0, but now it is V1.3.3.
